src/components/Charts/ModelChart.js

```diff
@@ -119,7 +119,7 @@ const ModelChart = ({
     },
   };
 
-  const [options] = useState({
```

Comment:
I'm not sure if this was some kind of "trick" or just a bug...  but as written, the chart wouldn't re-render when you adjusted the params passed in, since this state value was static and didn't update after initial render.

Reply:
@Fiskell you are familiar with this change it might affect the county level data

Reply:
@mikelehen It's a bug, we have it fixed on the counties branch and, @zjrosen1 We fixed it on the branch Jeremy was working on.  I thought that was out, is this branch behind develop?

Reply:
It's not fixed in develop: https://github.com/covid-projections/covid-projections/blob/adf258e017d64b3043f24ce92d605b38fbded3ab/src/components/Charts/ModelChart.js#L128

But looks like it is fixed in counties-model-page (https://github.com/covid-projections/covid-projections/blob/fe5497f5d0c89a50985771a36e6fcf07783dcf7c/src/components/Chart/Chart.js#L244).

I didn't know about that branch (I saw a bunch of county-related stuff in develop, so I thought the county work was [largely] merged).  Do we know when it's going to land?  I don't want to create merge conflicts for it...